packages/webapp/pages/posts/[id]/similar.tsx

```diff
@@ -1,146 +0,0 @@
-import AuthContext from '@dailydotdev/shared/src/contexts/AuthContext';
-import {
-  POST_BY_ID_STATIC_FIELDS_QUERY,
-  PostData,
-  PostType,
-} from '@dailydotdev/shared/src/graphql/posts';
-import { OtherFeedPage } from '@dailydotdev/shared/src/lib/query';
-import { SIMILAR_POSTS_FEED_QUERY } from '@dailydotdev/shared/src/graphql/feed';
-import Feed from '@dailydotdev/shared/src/components/Feed';
-import {
-  FeedPageHeader,
-  truncateTextClassNames,
-} from '@dailydotdev/shared/src/components/utilities';
-import {
-  GetStaticPathsResult,
-  GetStaticPropsContext,
-  GetStaticPropsResult,
-} from 'next';
-import request, { ClientError } from 'graphql-request';
-import { graphqlUrl } from '@dailydotdev/shared/src/lib/config';
-import { ApiError } from '@dailydotdev/shared/src/graphql/common';
-import usePostById from '@dailydotdev/shared/src/hooks/usePostById';
-import SimilarEmptyScreen from '@dailydotdev/shared/src/components/feeds/SimilarEmptyScreen';
-import { GoBackHeaderMobile } from '@dailydotdev/shared/src/components/post/GoBackHeaderMobile';
-import classNames from 'classnames';
-import { getLayout } from '../../../components/layouts/FeedLayout';
-import { mainFeedLayoutProps } from '../../../components/layouts/MainFeedPage';
-import { defaultOpenGraph, defaultSeo } from '../../../next-seo';
-import Custom404 from '../../404';
-
-export interface Props {
-  id: string;
-  initialData?: PostData;
-}
-
-const SimilarFeed = ({ id, initialData }: Props): ReactElement => {
-  const { user } = useContext(AuthContext);
-  const { post, isError } = usePostById({
-    id,
-    options: { initialData, retry: false },
-  });
-  const { FeedPageLayoutComponent } = useFeedLayout();
-  // Must be memoized to prevent refreshing the feed
-  const queryVariables = useMemo(
-    () => ({
-      postId: id,
-      supportedTypes: [
-        PostType.Article,
-        PostType.VideoYouTube,
-        PostType.Collection,
-      ],
-    }),
-    [id],
-  );
-
-  const seo: NextSeoProps = {
-    title: `Similar posts to ${post?.title} on daily.dev`,
-    openGraph: { ...defaultOpenGraph },
-    ...defaultSeo,
-  };
-
-  if (isError) {
-    return <Custom404 />;
-  }
-
-  const Title = () => (
-    <h1
-      className={classNames('font-bold typo-callout', truncateTextClassNames)}
-    >
-      Posts similar to &quot;{post?.title}&quot;
-    </h1>
-  );
-
-  return (
-    <>
-      <GoBackHeaderMobile>
-        <Title />
-      </GoBackHeaderMobile>
-      <FeedPageLayoutComponent className="overflow-x-hidden">
-        <NextSeo {...seo} />
-        <Feed
-          header={
-            <FeedPageHeader className="mb-5">
-              <Title />
-            </FeedPageHeader>
-          }
-          feedName={OtherFeedPage.SimilarPosts}
-          feedQueryKey={[
-            'similarPostsFeed',
-            user?.id ?? 'anonymous',
-            Object.values(queryVariables),
-          ]}
-          query={id && SIMILAR_POSTS_FEED_QUERY}
```

Comment:
We can also remove `SIMILAR_POSTS_FEED_QUERY`?

Reply:
@capJavert Yep, also cleanup up some other feed related things now :)